py/desispec/scripts/qsoqn.py

```diff
@@ -209,55 +210,61 @@ def extract_redshift_info_from_RR(filename_redrock, targetid):
     # make an independant run for each quasar templates to circumvent some problem from redrock:
     # 1] cannot give two templates in redrock as input, only one or a directory
     # 2] problem with prior and template redshift range .. --> give zero result and redrock stop
-    for template_filename in param_RR['templates_filename']:
-        filename_priors = param_RR['filename_priors']
-        filename_output_rerun_RR = param_RR['filename_output_rerun_RR']
-        filename_redrock_rerun_RR = param_RR['filename_redrock_rerun_RR']
 
-        # find redshift range of the template
+    filename_priors = param_RR['filename_priors']
+    filename_output_rerun_RR = param_RR['filename_output_rerun_RR']
+    filename_redrock_rerun_RR = param_RR['filename_redrock_rerun_RR']
+
+    # find redshift range spanned by templates
+    zmin = 100
+    zmax = -100
+    for template_filename in param_RR['templates_filename']:
```

Comment:
If the ship has sailed, that's okay, but it's a little odd to me that this has a singular name but can be a list of multiple files.

Reply:
That naming scheme was leftover from the original code, but given that this is only used internally to the afterburner (not a public API), I updated "templates_filename" to "template_filenames" throughout.